modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/validation/ParameterTypeModelValidator.java

```diff
@@ -132,6 +148,18 @@ private void checkInvalidFieldAnnotations(NamedObject model, String typeName, Fi
           }
         }
       }
+
+      private boolean isPojoWithoutDefaultConstructor(Class<?> clazz) {
+        return (!clazz.isInterface() && !isAbstract(clazz.getModifiers())
+            && Stream.of(clazz.getConstructors())
+                .noneMatch(c -> c.getParameterCount() == 0));
+      }
+
+      private Class<?> getTypedValueGenericClass(Type type) {
+        return !CollectionUtils.isEmpty(type.getGenerics())
+            ? type.getGenerics().stream().findFirst().get().getConcreteType().getDeclaringClass().get()
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Call "Optional#isPresent()" before accessing the value. ([squid:S3655](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS3655))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7615&issues=AWhykiEpotIXnmsUzH_b&open=AWhykiEpotIXnmsUzH_b)

[ISSUE_KEY]: AWhykiEpotIXnmsUzH_b